app/assets/stylesheets/group.scss

```diff
@@ -0,0 +1,48 @@
+$white: #ffffff;
+
+.title {
+  color: $white;
+  font-size: 60px;
+  font-size: 5vw;
+  font-weight: bold;
+}
+.groups {
+  text-align: center;
+}
+
+.group {
+  margin: 20px;
+  padding: 30px;
+  background-color: rgba(244, 162, 23, 0.66);
+  max-width: 1000px;
+  -moz-border-radius: 15px;
+  -webkit-border-radius: 15px;
+  border-radius: 15px;
+}
+
+.group-title {
+  font-size: 25px;
```

Comment:
Properties should be ordered background-color, -moz-border-radius, -webkit-border-radius, border-radius, color, font-size, font-size, padding